client/src/utils/dates/Date.js

```diff
@@ -1,8 +1,19 @@
-export function isDate() {
-  const today = new Date();
-  const date = `${today.getDate()}.${
-    today.getMonth() + 1
-  }.${today.getFullYear()}`;
-
-  return date;
-}
+// Unix Timestamps
+export const isUnixTime = () => Math.floor(Date.now() / 1000);
+
+// date-format: DD.MM.YYYY
+export const isDate = (today = new Date()) =>
+  `${("0" + today.getDate()).slice(-2)}.${("0" + (today.getMonth() + 1)).slice(
+    -2
+  )}.${today.getFullYear()}`;
+
+// time-format: HH:MM
+export const isTime = (time = new Date()) =>
+  `${("0" + time.getHours()).slice(-2)}.${("0" + time.getMinutes()).slice(-2)}`;
```

Comment:
```suggestion
export const isTime = (time = new Date().toLocaleTimeString("de-DE").substr(0, 5)
```

This should do the same as you did there.

Reply:
the answer above